Qausim/PropertyPro-lite
# #167587483 Connect UI View All Properties Implementation to Server

#### What does this PR do
Connects web page to the endpoint to view fetch all property adverts

#### Description of Task to be completed?
Fetch all properties adverts from **GET** `/api/v1/property` and render on the page

#### How should this be manually tested?
Clone this repo, cd into it and 
- Run `npm run server`
- Open `signin.html` or `signup.html` in a browser and supply valid credentials to see the `properties.html`

#### Any background context you want to provide?
User should be able to view server-provided property adverts

#### What are the relevant pivotal tracker stories?
[#167587483](https://www.pivotaltracker.com/story/show/167587483)

#### Screenshots (if appropriate)
None

#### Questions:
None
